packages/ui/src/components/DateInput/__stories__/Size.stories.tsx

```diff
@@ -9,3 +9,7 @@ export const Size: StoryFn<typeof DateInput> = args => (
     ))}
   </Stack>
 )
+
+Size.args = {
```

Comment:
I think you need to add a decorators for this story with a div that has a bigger height or we won't see correctly the dropdown

![Screenshot 2024-11-13 at 11 06 02](https://github.com/user-attachments/assets/d1c90bac-804c-4803-a7d6-79ced9a6d7b5)